backend/src/main/java/jonaszeihe/ninjin/controller/ElementController.java

```diff
@@ -0,0 +1,58 @@
+package jonaszeihe.ninjin.controller;
+
+
+import jonaszeihe.ninjin.model.*;
+import jonaszeihe.ninjin.service.ElementService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("api/element")
+public class ElementController {
+
+    private final ElementService elementService;
+
+    @Autowired
+    public ElementController(ElementService elementService) {
+        this.elementService = elementService;
+    }
+
+    @PostMapping
+    public void addElements(@RequestBody AddElementDto dto) {
+        this.elementService.addElements(dto.getCourseName(), dto.getElementContent(), dto.getCourseSize());    }
+
+    @GetMapping("{segmentName}")
+    public List<Element> listElementsBySegmentName(@PathVariable String segmentName) {
+        return elementService.listElementsBySegmentName(segmentName);
+    }
+
+    @GetMapping("{elementName}")
+    public Element getElementByElementName(@PathVariable String elementName) {
+        return elementService.getElementByElementName(elementName)
+                    .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Element not found"));
+
+    }
+
+    @PutMapping("/{elementName}")
```

Comment:
```suggestion
    @PutMapping("{elementName}")
```
add a slash for all or remove them all ;)